### DISCUSSION TITLE
Hope to change the type_map after loading the trained model by checkpoint

### DISCUSSION THREAD

**Discussion by wei0py** [deepmodeling/deepmd-kit#1701] [Q&A]
Hi,

is it possible to change the type_map(like adding another element type) after loading the previous training?
e.g. I have a mode for Fe, now I want to add some Fe-Al data and continue training based on the model for Fe.

Thanks!

**Comment by AnguseZhang** [comment]
As far as I know, we currently do not support such circumstance. Probably you should start another training from scratch.

**Reply by wei0py** [reply]
Is it possible to manually edit a checkpoint and inherent the parameters for one type from previous model?
